react-version/src/components/game/BoardButtons.js

```diff
@@ -16,8 +16,20 @@ import ButtonsRow14 from "./buttons/ButtonsRow14";
 import ButtonsRow15 from "./buttons/ButtonsRow15";
 import ButtonsRow16 from "./buttons/ButtonsRow16";
 import buttonsData from "./buttons/buttonsData/buttonsData";
+import { BoardContext } from "../context/BoardContext";
 
 const BoardButtons = (props) => {
+	const [board, setBoard] = useContext(BoardContext);
+	const handleArrowButton = (event) => {
+		props.onArrowButton();
+		const handleBtnId = (e) => {
+			setBoard({
+				...board,
+				btnId: e.target.id,
+			});
+		};
+		handleBtnId(event);
```

Comment:
We use `handleBtnId` function only once, what this extra function definition gives us (instead of more code)?

Reply:
this function must be called everytime player want to move character to the next hex due to assign active button (where character stands).